sheetScript.gs

```diff
@@ -495,6 +498,26 @@ function deletePlaylistItems(playlistId, deleteBeforeTimestamp) {
       nextPageToken = null;
     }
   }
+
+  // Delete Duplicates Videos by videoId
+  try {
+    let tempVideos = [];
+    let duplicateVideos = [];
+
+    allVideos.forEach(x => {
+      if (tempVideos.find(y => y.contentDetails.videoId === x.contentDetails.videoId)) {
```

Comment:
This could be made more efficient using a hashmap, and could probably be merged into the previous for loop like `else if (removeDuplicatesFlag && seenVideosMap.has(item.id)) delete`.

Reply:
sorry I don't know how to do it

Reply:
that's fine, I'll just modify it later